src/modules/specialEvents/SpecialEvent.ts

```diff
@@ -23,6 +24,17 @@ export default class SpecialEvent {
     // TODO: only notify once initially until event about to start/end
     notified: SpecialEventNotifiedStatus;
 
+    disabledSpecialEvents = Settings.getSetting('disableSpecialEvents').observableValue.subscribe((newValue) => {
```

Comment:
You are currently setting the value to the result of a void function. Events are not working because of this